CMakeLists.txt

```diff
@@ -75,9 +75,9 @@ set(SELFHOST_SOURCES
   selfhost/ide.jakt
```

Comment:
At this point stage0 will not know about `comptime`
So when bisecting this commit would fail

Reply:
There's not much else I can do besides squashing the two commits.